### ISSUE TITLE
[Master Issue] Optimize the classification algorithm

### ISSUE THREAD

**Issue by will-r-wang** [will-r-wang/envirolization#7]
Once we have a basic neural net working, we need to make serious optimizations (tweaking data, params, features, learning rate, etc) to ensure that the accuracy is at least somewhere reasonable (60%+).
